packages/sdk-react/src/components/approvalPolicies/ApprovalPolicyDetails/useApprovalPolicyDetails.tsx

```diff
@@ -26,10 +27,42 @@ export const useApprovalPolicyDetails = ({
   onCreated,
   onUpdated,
 }: Props) => {
-  const createMutation = useCreateApprovalPolicy();
-  const updateMutation = useUpdateApprovalPolicy();
+  const { i18n } = useLingui();
+  const { api, queryClient } = useMoniteContext();
+
+  const createMutation = api.approvalPolicies.postApprovalPolicies.useMutation(
+    {},
+    {
+      onSuccess: async () => {
+        await api.approvalPolicies.getApprovalPolicies.invalidateQueries(
```

Comment:
While you migrated the hook absolutely right, I spotted the error that the detailed approval policy data was not invalidated. The error was in the previous hooks as well.
Could you add the `api.approvalPolicies.getApprovalPoliciesId` invalidation? 
```
await api.approvalPolicies.getApprovalPoliciesId.invalidateQueries(
          {
            parameters: { path: { approval_policy_id: approvalPolicy.id } },
          },
          queryClient
        );
```
The same for `updateMutation`

Reply:
Thank you, such a nice catch, fixed.